definitions/variable/industry/scrap.yaml

```diff
@@ -0,0 +1,32 @@
+
+- Collected Scrap|Iron and Steel|Steel|{Demand Sector}:
+    description: Steel scrap from {Demand Sector}
+    unit: Mt/yr
+- Collected Scrap|Non-Metallic Minerals|{Non-Metallic Minerals Commodity}|{Demand Sector}:
```

Comment:
Actually, scrap is a terminology used for metals but not so much for other materials. We can either use it just for metals or to be consistent across all materials, we could adopt the terminology "waste" which can cover metals, non-metallic minerals, plastics, and others. In this case "Collected Waste" might sound confusing. We can have something like "Total Waste|xxxx", and "Recycled Waste|xxxx".

Reply:
Separating these variables into Scrap vs Waste and call it Total (with an extended description, please) vs. Recycled makes a lot of sense to me, thanks!